kebechet/managers/version/version.py

```diff
@@ -75,6 +76,8 @@
 _EVENTS_SUPPORTED = ["issues", "issue"]
 # Maximum number of log messages in a single release. Set due to ultrahook limits.
 _MAX_CHANELOG_SIZE = 300
+# Previous release tag present
+_PREV_RELEASE_TAG = False
```

Comment:
Can this be moved to the class as an instance variable rather than using a global?

Reply:
They are being used in one of the static methods so I chose to use, we could change `_compute_changelog` to a class method and then maybe add it as an instance variable? Is that a good idea?

Reply:
Yes, I think it will be better than a global variable. If it would make sense, the static method can become an instance method.

Reply:
Changed :)